content/docs/command-reference/exp/save.md

```diff
@@ -16,6 +16,10 @@ See also `dvc exp run`, which includes this operation.
 usage: dvc exp save [-h] [-q | -v] [-f]
                    [--json] [-n <name>]
                    [-I path] [-m <message>]
+                   [targets [targets ...]]
+
+positional arguments:
+  targets          Stages to save. 'dvc.yaml' by default.
```

Comment:
(just reading this for the first time, and not reading the initial context)

I can't come with an explanation in my head on what it does and why we need this tbh :) Probably means we need to provide a bit more context in the description and/or change the message a bit to be more explicit (?)

Reply:
Changed the text and added an example